Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -362,11 +362,11 @@ goToStmt : GOTO whiteSpace expression;
 ifStmt :
      IF whiteSpace booleanExpression whiteSpace THEN endOfStatement
      block
-     elseIfBlock*
-     elseBlock?
-     END_IF
+     (statementLabelDefinition? whiteSpace? elseIfBlock)*
+     (statementLabelDefinition? whiteSpace? elseBlock?)
+     statementLabelDefinition? whiteSpace? END_IF
```

Comment:
This is where my first comment should have gone. The problem I see here is that the line number is not part of the elseif block or else block. That might cause problems with the quickfixes.